sig-rktnetes/README.md

```diff
@@ -0,0 +1,17 @@
+# The Mission
+`sig-rktnetes` is focused on the topics related to Kubernetes integration of container runtime rkt which is referred to as rktnetes. 
+
+This includes:
+- Kubernetes and rkt integration
+- Performance and scalability
+- Images, package management
+- Rktnetes deployments
+- Issues related to monitoring (cAdvsor, Heapster, etc.)
+
+The main objective is to maintain and develop the integration of rkt to the Kubernetes as the container runtime.
```

Comment:
I think we can make this a little more ambitious and/or clear.

"The main objective of this SIG is to ensure that rkt is the best runtime to use with a Kubernetes cluster" perhaps?